tests/integration/operator/main_test.go

```diff
@@ -24,7 +24,6 @@ import (
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite("operator_controller", m).
-		Skip("https://github.com/istio/istio/issues/22969").
```

Comment:
Can we tweak the makefile (temporarily, not to merge) to run what we run in postsubmit here before we merge this again?